supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc

```diff
@@ -0,0 +1,245 @@
+[[red-hat-directory-server-conventions]]
+
+For documentation questions, contact rhel-docs@redhat.com.
+
+[discrete]
+[[access-control-instruction]]
+==== image:images/yes.png[yes] access control instruction (noun)
+*Description*: In an LDAP directory, an access control instruction (ACI) grants or denies access to entries or attributes. Use "access control instruction" on the first usage and then the abbreviation "ACI" subsequently.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[attribute]]
+==== image:images/yes.png[yes] attribute (noun)
+*Description*: Each entry in an LDAP directory contains attributes. Object classes in an entry control which attributes in an entry are optional and which are required.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[base-dn]]
+==== image:images/yes.png[yes] base DN (noun)
+*Description*: In an LDAP directory, the base distinguished name (DN) defines the starting point for operations, such as searches.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:distinguished-name[distinguished name]
+
+[discrete]
+[[bind-dn]]
+==== image:images/yes.png[yes] bind DN (noun)
+*Description*: A distinguished name (DN) defines the unique location of an entry in the LDAP directory. You can use the DN of an entry to bind (authenticate) to an LDAP directory. The bind DN is similar to a user name in other systems.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:distinguished-name[distinguished name]
+
+[discrete]
+[[consumer-rhds]]
+==== image:images/yes.png[yes] consumer (noun)
+*Description*: In an LDAP replication environment, consumers receive data from suppliers or hubs.
+
+*Use it*: yes
+
+*Incorrect forms*: slave
+
+*See also*: xref:hub[hub], xref:supplier[supplier]
+
+[discrete]
+[[directory-manager]]
+==== image:images/yes.png[yes] Directory Manager (noun)
+*Description*: The privileged administrative user in Red Hat Directory Server is called Directory Manager. The distinguished name (DN) of this user is cn=Directory Manager.
+
+*Use it*: yes
+
+*Incorrect forms*: DM, directory manager
+
+*See also*:
+
+[discrete]
+[[directory-server]]
+==== image:images/yes.png[yes] Directory Server (noun)
+*Description*: The short product name of Red Hat Directory Server. In the title of guides, use the full product name "Red Hat Directory Server" and, elsewhere, the short name "Directory Server". Because it is the product name, both words start with a capital letter. Additionally, this distinguishes the Red Hat Directory Server product from other directory servers.
```

Comment:
```suggestion
*Description*: The short product name of Red Hat Directory Server. In the title of guides, use the full product name "Red Hat Directory Server" and, elsewhere, the short name "Directory Server". Because it is the product name, both words start with a capital letter. Additionally, this practice distinguishes the Red Hat Directory Server product from other directory servers.
```
Consider adding a pronoun reference. It's ambiguous what "this" refers to.

Reply:
Fixed